raiden/messages.py

```diff
@@ -382,14 +365,21 @@ class Processed(SignedRetrieableMessage):
     """
 
     # FIXME: Processed should _not_ be SignedRetrieableMessage, but only SignedMessage
-    cmdid: ClassVar[int] = messages.PROCESSED
+    cmdid: ClassVar[CmdId] = CmdId.PROCESSED
 
     message_identifier: MessageID
 
     @classmethod
     def from_event(cls, event):
         return cls(message_identifier=event.message_identifier, signature=EMPTY_SIGNATURE)
 
+    def _data_to_sign(self) -> bytes:
+        return pack_data(
+            (self.cmdid.value, "uint8"),
+            (b"\x00" * 3, "bytes"),  # padding
```

Comment:
There are a lot of these, can we make the `cmdid` a `uint32`?

Reply:
We can, but I wanted to avoid changing the message format in this PR. Depending on the purpose if `cmdid`, you could:
* remove both `cmdid` and padding
* remove padding
* keep as is
* make `cmdid` a uint32 or larger

https://github.com/raiden-network/raiden/issues/4297 needs cleared up before doing any of these.

Reply:
Please, just leave it be, there is no clear benefit on removing this ...  by now we are removing things just for the sake of it, this may break important properties, like replay attack protection.